src/volur/sdk/v1alpha2/client.py

```diff
@@ -4,6 +4,7 @@
 from loguru import logger
 from volur.api.v1alpha1.client import VolurApiAsyncClient
 from volur.sdk.v1alpha1.sources.csv.base import MaterialsSource
+from volur.sdk.v1alpha2.sources.csv.base import ProductsSource
```

Comment:
Both these lines could use the structure you are buidling, which is absolutely correct:

 ```suggestion
from volur.sdk.v1alpha1.sources.csv import MaterialsSource
from volur.sdk.v1alpha2.sources.csv import ProductsSource
```

Reply:
Updated when merged with main